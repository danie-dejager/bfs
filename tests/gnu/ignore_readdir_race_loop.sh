# Make sure -ignore_readdir_race doesn't suppress ELOOP from an actual filesystem loop
! bfs_diff -L loops -ignore_readdir_race
