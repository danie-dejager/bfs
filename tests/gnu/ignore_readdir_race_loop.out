loops
loops/broken
loops/deeply
loops/deeply/nested
loops/deeply/nested/dir
loops/file
loops/notdir
loops/skip
loops/skip/dir
loops/skip/loop
loops/symlink
